haskell/ghc_bindist.bzl

```diff
@@ -343,9 +343,16 @@ def _ghc_bindist_impl(ctx):
     )
 
     if os == "windows":
+        # These libraries cause linking errors on Windows when linking
+        # pthreads, due to libwinpthread-1.dll not being loaded. It's
+        # hard to guesss the paths of these libraries, so we have to
+        # use dir to recursively find them.
+        result = ctx.execute(["cmd", "/c", "dir", "/s", "/b", "libstdc++.dll.a"])
+        for path in result.stdout.splitlines():
+            ctx.execute(["cmd", "/c", "del", path.strip()])
+
         # These libraries cause linking errors on Windows when linking
         # pthreads, due to libwinpthread-1.dll not being loaded.
-        execute_or_fail_loudly(ctx, ["rm", "mingw/lib/gcc/x86_64-w64-mingw32/7.2.0/libstdc++.dll.a"], working_directory = unpack_dir)
         execute_or_fail_loudly(ctx, ["rm", "mingw/x86_64-w64-mingw32/lib/libpthread.dll.a"], working_directory = unpack_dir)
         execute_or_fail_loudly(ctx, ["rm", "mingw/x86_64-w64-mingw32/lib/libwinpthread.dll.a"], working_directory = unpack_dir)
 
```

Comment:
Good point, yes. I think in those cases the path easier to predict because it didn't contain that mingw version. But, it makes sense to be consistent.